mempool/src/shared_mempool.rs

```diff
@@ -406,10 +408,14 @@ where
         subscribers,
     };
 
-    let interval =
-        timer.unwrap_or_else(|| default_timer(config.mempool.shared_mempool_tick_interval_ms));
+    let interval_ms = config.mempool.shared_mempool_tick_interval_ms;
+    let smp_outbound = smp.clone();
+    let f = async move {
+        let interval = timer.unwrap_or_else(|| default_timer(interval_ms));
+        outbound_sync_task(smp_outbound, interval).await
+    };
```

Comment:
this is needed because we can't construct the timer outside of a tokio context, right?